index.bs

```diff
@@ -556,8 +563,34 @@ To <dfn>process an attribution source</dfn> given an [=attribution source=] |sou
 1. [=list/Remove=] all entries in |cache| where the entry's [=attribution source/expiry time=] is less than the current time.
 1. If the [=list/size=] of |cache| is greater than or equal to the user agent's
      [=max source cache size=], return.
+1. If |source|'s [=attribution source/attribution mode=] is "<code>falsely</code>", then:
+    1. Assert: |source|'s [=attribution source/source type=] is "<code>event</code>".
+    1. Let |fakeReport| be a new [=attribution report=] with the items:
```

Comment:
Should this pass in a generated trigger to obtain a report instead?

Reply:
One issue is that a trigger requires a trigger origin, but we only have an attribution destination (site) from the source. I guess we could change the trigger structure to store a site instead?

Reply:
Looks like trigger origin is used only used an origin for data clearing... which is erroneous anyway because triggers are not stored in any of the caches. Assuming that behavior was fixed to pass the origin to the report, we would actually still require a trigger origin here.

Perhaps we could add an attribution destination to triggers (in addition to the origin), and set the trigger origin as an opaque origin which should not break any assertions in this spec. wdyt?